scripts/cpu/name.sh

```diff
@@ -1,4 +0,0 @@
-#!/bin/bash
-
```

Comment:
Can you leave those two scripts as examples, we can remove them later, I can modify the output to match the protocol if you don't want to

Reply:
I was planning a more sophisticated cpu support as a follow-up pull request...